### PR TITLE
Add dc contributor

### PR DESCRIPTION
**Pull request by brinxmat**
Adds mapping  dc.contributor -> contributor object

### CODE REVIEW

File: import-brage/src/main/java/no/unit/nva/importbrage/DcValue.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 
-
+@SuppressWarnings("PMD.UnusedPrivateField")
```
Comment (janbj) [inline import-brage/src/main/java/no/unit/nva/importbrage/DcValue.java:10]: What do you think about always disabling PMD rules in config instead?

File: import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    private ContributorType contributorType;
+    private String value;
+
+
+    public BrageContributor(DcValue value) {
+        this(value.getQualifier(), value.getValue());
+    }
+
+    public BrageContributor(String qualifier, String value) {
+        this(ContributorType.getTypeByName(qualifier), StringEscapeUtils.unescapeHtml4(value));
```
Comment (janbj) [inline import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java]: Why do you escape here and not in the constructor you are calling?

File: import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java [hunk 3]
```diff
@@ -1,36 +1,26 @@
...
     void xmlImportLoadsData() throws IOException {
-        var dublinCore = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
-        var values = dublinCore.getDcValues();
-        values.forEach(this::testDcValue);
+        var contributors = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
+        contributors.forEach(this::testContributor);
     }
 
-    private void testDcValue(DcValue value) {
+    private void testContributor(BrageContributor value) {
```
Comment (janbj) [inline import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java:29]: asstertContributor or verifyContributor?

File: import-brage/src/main/java/no/unit/nva/importbrage/XmlImport.java [hunk 4]
```diff
@@ -2,19 +2,33 @@
...
 
-    public static DublinCore map(File file) throws IOException {
-        return objectMapper.readValue(file, DublinCore.class);
+    /**
+     * Returns a list of contributors from a Brage qualified Dublin Core xml export.
+     * @param file An XML file.
+     * @return A list of contributor objects.
+     * @throws IOException If the file cannot be found.
+     */
+    public static List<BrageContributor> map(File file) throws IOException {
```
Comment (janbj) [inline import-brage/src/main/java/no/unit/nva/importbrage/XmlImport.java]: Rename method to getContributors?

Comment (axthosarouris) [reply import-brage/src/main/java/no/unit/nva/importbrage/XmlImport.java]: extractContributors.

File: import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+          dc.contributor.advisor        Use primarily for thesis advisor.
+          dc.contributor.author         Use for author.
+          dc.contributor.department     The department where an item was created (used for import from DiVA).
+          dc.contributor.editor         Use for editor.
+          dc.contributor.illustrator    Use for illustrator.
+          dc.contributor.orcid          Use for Orcid identifier of a contributor
+          dc.contributor.other
+     */
+
+    private ContributorType contributorType;
```
Comment (axthosarouris) [inline import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java]: final?

File: import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+    public ContributorType getContributorType() {
+        return contributorType;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public enum ContributorType {
```
Comment (axthosarouris) [inline import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java:52]: Consider putting the enum is its own file.

File: import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+          dc.contributor.author         Use for author.
+          dc.contributor.department     The department where an item was created (used for import from DiVA).
+          dc.contributor.editor         Use for editor.
+          dc.contributor.illustrator    Use for illustrator.
+          dc.contributor.orcid          Use for Orcid identifier of a contributor
+          dc.contributor.other
+     */
+
+    private ContributorType contributorType;
+    private String value;
```
Comment (axthosarouris) [inline import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java]: Is this the name of the contributor?

File: import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java [hunk 8]
```diff
@@ -1,36 +1,26 @@
...
 
     public static final String OVERCOMPLETE_EXAMPLE = "/overcomplete_example.xml";
-    public static final Set<String> LANGUAGE_VALUES = Set.of("*", "en", "en_US");
 
     @Test
     void xmlImportLoadsData() throws IOException {
-        var dublinCore = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
-        var values = dublinCore.getDcValues();
-        values.forEach(this::testDcValue);
+        var contributors = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
```
Comment (axthosarouris) [inline import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java]: Create a file that has one entry and verfify that the data in the entry are as expected

File: import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java [hunk 9]
```diff
@@ -1,36 +1,26 @@
...
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class XmlImportTest {
 
     public static final String OVERCOMPLETE_EXAMPLE = "/overcomplete_example.xml";
-    public static final Set<String> LANGUAGE_VALUES = Set.of("*", "en", "en_US");
 
     @Test
     void xmlImportLoadsData() throws IOException {
```
Comment (axthosarouris) [inline import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java:18]: One test is xmlImportReturnsAListOfContributorsWhenInputIsFileContainingContributors and another test could be 
xmlImportDoesNotCreateContributorsFromElementsThatAreNotContributors

File: import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java (hunk 3 above)
Comment (axthosarouris) [reply import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java:29]: assertThatContributorsHaveTypeAndName

File: import-brage/src/main/java/no/unit/nva/importbrage/DcValue.java (hunk 1 above)
Comment (brinxmat) [reply import-brage/src/main/java/no/unit/nva/importbrage/DcValue.java:10]: I'm hoping that these fields are the exception

File: import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java (hunk 2 above)
Comment (brinxmat) [reply import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageContributor.java]: I don't think it is necessary to unescape at all, I will check

File: import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java (hunk 3 above)
Comment (brinxmat) [reply import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java:29]: These tests will be replaced in the next revision

File: import-brage/src/main/java/no/unit/nva/importbrage/XmlImport.java (hunk 4 above)
Comment (brinxmat) [reply import-brage/src/main/java/no/unit/nva/importbrage/XmlImport.java]: See following commits

**Pull request merged**